### ISSUE TITLE
authedmine again

### ISSUE THREAD

**Issue by Leseratte10** [easylist/easylist#710]
I am terribly sorry to open another issue related to authedmine and #706 but I just don't understand why this issue has been closed again. 

You asked for examples. 
- "FireServ" then said he operates a small game giving users ingame points for mining
- "loadedlen" said he operates a charity website and uses authedmine to collect money
- "gpaulius" said he operates a web site giving users points for mining
- "ICTman1076" said he uses this instead of advertisement to support his web page
- I added an example to use this instead of captchas, although I haven't implemented this yet on a web page. 
- "bitcoins43" posted his example page in another issue ( #707 ) and it was closed immediately ... -> http://www.bitcoins43.com/miner/
- "DatDraggy" posted a link to his page using authedmine for support below -> https://www.kieran.pw/ 
- Here is a link to a blog which did a test phase with authedmine but disabled it again later -> http://nuclearcryptobuddha.blog/2017/10/coinhive-javascript-monero-miner/ 
- The "mouse5.io" forum supports the authedMine scripts as a donation variant, although only for logged-in users: https://mouse5.io/forum/index.php?threads/free-donation-through-cryptomining.1700/ 
- This paperclip online minigame uses it instead of ads: https://paperclipsgame.com/no-ads-policy 
- The german blog "bitpage" uses it instead of ads: https://bitpage.de/unterstuetze-uns/
- and many more examples listed in comments to this issue (collecting them all here so its easier for you to find): http://hydromining.info/dashboard/register.php | http://amf.lt/register/ | https://www.scamadviser.com/report-form.php | https://blog.jkanetwork.com/post.php?idpost=64 | https://soundpark.red/registration | http://monero.price.exchange/ | https://davidyat.es/ | https://nikonpc.com/ | https://www.yuzde100yerli.com/ | https://www.schwarze-rose.cc/faq#coinhive-statt-werbung | https://www.videosharp.info/project/id=678 | http://1320challenge.com/ | https://cpuforgood.org/ | https://github.com/ctubio/Krypto-trading-bot | https://l.ehehdada.com/ | https://vivebtc.win/mining/btc | http://superfaucet.xyz

These are 28 web pages actively using authedmine, either the standard miner to gain money, or the captcha variant, which are all broken by this filter. Still need more examples? The other issue has been opened for less than four hours. Maybe would it be possible to leave an issue like this open for a longer time, like a week or two, and not close it after a few hours just because you haven't gotten the reply you wanted? People could discuss this, people would probably also post more example links. But not if you close an issue almost immediately. 

And even if this technique would not be in use yet right now - which it is - why would that change anything about blocking or unblocking it? It is a very new technique. Why do you expect hundreds of website owners to already use this? How / why do you expect website owners to implement this *at all* if half of the users can't use it because of their ad blocker?

This is a new technique which allows website owners to stop using ads. Isn't that the point of an adblocker? Then why work against something like that? Just because there are no examples of this in use yet?

**Please** reconsider your opinion on blocking this, or, at least, leave this issue open for a while for discussing and not close it immediately again. 

The amount of issues opened about this and the amount of comments and likes on these issue should tell you that this is block list entry is a problem and should be removed. Other than EasyList members, nobody of the commenters requested this page to stay blocked. Please leave this issue open, so this can be discussed, and not just close it again after a few hours or less.

**Comment by DatDraggy** [comment]
Somebody asked for an example so I put one of the authedmine scripts on my index. https://www.kieran.pw
(It's also on https://authedmine.com/ but I guess the person didn't bother reading or I'm not understanding what they mean by "an example")

**Comment by bitcoins43** [comment]
I posted my link before (#707) and also gave instructions to test it. http://www.bitcoins43.com/miner/
Issue was simply closed with no reply.

**Comment by loadedlen** [comment]
You're welcome to try out the miner on my site, though you will need to create a login first. The address is https://coiners.club the charity aspect shares equally in a side sweepstakes (this is an incentive for people to stay on the page) right now, the page uses coinhive with code to detect adblockers. I WAS using authedmine but it got blocked, so I didn't see a reason for multiple disclaimers and opt-ins if the user had to whitelist the domain as well. My disclosure is clearly visible before the miner starts. It allows the user a choice, and is displayed everytime the page is accessed (no cookie)

**Comment by trank2012** [comment]
Don't worry guys, very soon Adblock will be in the same situaton, Google Chrome will ban it.

**Comment by FFormula** [comment]
I have my own programmer's videoschool [www.videosharp.info](https://www.videosharp.info/project/id=678)  (on russian page), and I startet to use Authedmine as chance to provide free videolessons for users, which can pay full price for courses. I think, It is very good idea, please, let Authedmine a chance for future!

**Comment by RedSQL** [comment]
Maintaining a site is the usual thing I do in every day life, and the authedmine miner is the only way I monetize my site, since adsense or any ads service isn't working out for me (I don't like ads myself, trust me). I actually got excited that authedmine is only with explicit opt-in from user-end and switched to that immediately as soon as I discovered that.
Not only I generated little money from people who actually wanted to support me, but now I receive no income at all because it has been blacklisted. I really don't understand whats the problem here, the miner requires explicit opt-in from user-end and it remembers the choice only for 24 hours, after that user have to accept or decline the miner again. 
Also remember, I'm not the only one who runs site like that, many people rely on their visitors running the miner so the owner can pay for servers, domain and etc.
Please, please, **PLEASE**, rethink the choice about blocking authedmine. Thank you.

**Comment by Castar** [comment]
Users get Information from website, and website get monero from users.
It's fair trade. right?

**Comment by enricodias** [comment]
The main reason for not blocking authedmine is because it's not and Ad and it's not a malware. By blocking it you are enforcing your personal opinion among others and causing trouble to the users and the websites. It's essentially censure.

By blocking contents like this, you are the malware.

**Comment by timporter** [comment]
Please don't block it, this is the change that online publishing needs.

**Comment by Pocketart** [comment]
I already explained in https://github.com/easylist/easylist/issues/655#event-1345849467 but he decided to delete my comment instead.

Easylist exists to deprive websites of their revenue.
Anything that allows a website to earn money they will block.

**Comment by kaptinboxxi** [comment]
I’ve had an optional miner on my site for CoinHive since Halloween. At peak so far, I hit 460 Hashes per second, average around 250 hashes per second. I added it as an option for those who couldn’t donate, ALSO I reward users with in-game currency for the game my site supports and that I am developing. I’ll eventually implement it into the game as well. I removed all ads as well

As of this second, Nov 16, 2017 at 7:35PM, Monero is at $121 per coin. As of right now, my site has generated 229.25 Million hashes for a total of 0.0344 XMR. That’s $4.16.

I believe this is the new, better way to go about all the ads on sites and such… give an opt out for ads via cpu mining.

My website is http://1320challenge.com

**Comment by nateritter** [comment]
I run a new project called https://cpuforgood.org which is dedicated to setting families of slaves free using the Authmined script. 

Please let innovation happen.

**Comment by ShiningBomb** [comment]
This is a revolutionary way to monetize without ads. Please let people try it first, and have feedback from them.

**Comment by Zhangsun321** [comment]
thank you for blocking ALL OF IT...

**Comment by ehehdada** [comment]
If it is useful as an example, at https://l.ehehdada.com we are using coinhive to let the user purchase premium features So it is a fair replacement of payment gateways like Stripe or PayPal for micropayments. And we use the captcha to protect the service from abuses of spammers.

**Comment by joseromerogc** [comment]
Hi I have 9 site, can see here: [(https://vivebtc.win/mining/btc)]. All paying direct a FaucetHub for every hash with price coinhive. I have >50000 visit daily.

**Comment by ShiningBomb** [comment]
Listen, if you succeed too kill coinhive, they might abandon and release them as open source and now everyone can run their own 'coinhive' with their own custom domain. No way adblock can stop this. This is different than ads that have to be centralized, coin mining is a decentralized game. The end game is on us whatever your action.

**Comment by thEpisode** [comment]
mmm what happend here? they guys ask for users if want to accept or not!, if not, nothing happend, NOTHING... is not intrusive, not in background, is controlled and it allow us to handle this users...

**Comment by golam9590** [comment]
nice

**Comment by Sharaykaka** [comment]
Hey  adblock  please don't block it....... It  is the biggest revolution ever in this field

**Comment by hbastidas** [comment]
this  ask for users if want to accept or not..

**Comment by NationsGlory** [comment]
Please don't block it, it's a revolution.

**Comment by Grathium** [comment]
This does not need to be blocked, it's not malware or an ad. By agreeing to participate in mining it is their own choice to use their CPU, don't make decisions for other people on your their behalf for them. Let them make their own decisions to help a creator who has spent time and effort into a site.

**Comment by strugmo** [comment]
Our site, SuperFaucet.Xyz, utlizes the the Javascript miner to allow users to earn bonuses and additional user levels.  We also use the Captcha for our faucet claim page.  Both the Captcha and the miner require users to voluntarily participate.  Neither starts automatically.  Nothing is stolen or taken from the user.  Our users gladly use the miner for both purposes listed above, but users who don't wish to participate can choose not to do so.

To block this technology wholesale is just flat out wrong.  This isn't about the mining technology. It's about bad actors operating web sites and building applications that automatically steal processing power from users using maligned or unfriendly miner technology.  

Block bad sites. Block or remove bad apps.  But to block an entire technology that is the biggest change in site monetizing in years seems hasty and unnecessary.

**Comment by Boses** [comment]
Authmine does not auto start which is why it was started as an alternative to coinhive meaning users are aware of the mining process through the GUI. Some websites use it for captcha and a way to get rid of ads. Doesn't make sense when your ad blocker blocks on of the ways websites are monetising  are trying to get rid of ads

**Comment by billyranario** [comment]
They should block the person who uses authedmine without the knowledge of their users. (sigh)

**Comment by Leseratte10** [comment]
You can't use authedmine without the user knowing, that is the point why blocking it is useless. The scripts on authedmine only run when you explicitly start them. 

The ones on coinhive can start without consent. And I have no problem with these being blocked. Authedmine on the other hand requires consent and should not be blocked.

**Comment by n0v3xx** [comment]
Unblock authedmine please! Best way to monetize projects in 2017!

**Comment by dhoogerwerf** [comment]
I plan to use Authedmine as alternative to a paywall for my site (i dont want to run ads, they are ugly) please dont block something that is a valid tool to battle ads on websites

**Comment by ryanbr** [comment]
**`"I use this for charity/purchases etc"`**

Suggest ABP/uBo users to whitelist the site, if mining is ***that*** important to you. 

**`"The script is user optional"`**

Basically we're putting trust in a mining company for this not to be changed/modified. I do not trust in any mining companys. 

**Note:**

Keep in mind these Mining server additions are not in Easylist (the main Adblock list), and only applies to users who specifically subscribed to Easyprivacy.

I'm happy to make a whitelist on specific hosted a legit captcha javascript. The only precondition the script isn't used for nothing mining related or tracking related. If its abused/changed to mine, it'll be removed.

Otherwise I don't see any reason to remove any of the filters.

**Comment by Leseratte10** [comment]
I don't get it. When I google "EasyPrivacy" and check the Easylist page, it states that "EasyPrivacy is an optional supplementary filter list that completely removes all forms of tracking from the internet, including web bugs, tracking scripts and information collectors, thereby protecting your personal data."

Authedmine does not track, is no web bug, is no tracking script and does not collect information. No personal data needs to be protected by this. 

Add authedMine to a new "no-mining" list if necessary, but please remove it from the EasyPrivacy list. Even if the EasyPrivacy list is not enabled by default, its description is inaccurate if things like this get added. If a user disables "tracking, web bugs and information collecting", this has nothing to do with voluntarily mining. 

I, as a user, would definitely start a mining script on a web page if they nicely ask me to do so, but I wouldn't necessarily add a whitelist entry for them because of more ads. And I don't think I'm the only one to think so either. 

The reason to remove this is basically: This is no tracking, does not disturb the users privacy, is no advertisement, and is voluntarily. It has nothing to do with a "privacy" black list. Add it to a "no-mining-list" if needed, but please remove it from the privacy list. 

Yes, you are putting trust in a mining company. So what? You are putting trust in thousands of other web pages by not putting them on a blacklist. They initially had a problem with people abusing the scripts - not because of coinhive but because people were using coinhive wrong - and then immediately took countermeasures to prevent this. Why is there a need to distrust them? It's not like they were lying about the consent part ...

**Comment by ryanbr** [comment]
Forwarding your friends/clients to this issue report, doesn't make it valid.  As stated, the filters are staying put.
